rhel/vex/parser.go

```diff
@@ -331,9 +353,58 @@ func (c *creator) lookupVulnerability(vulnKey string, protoVulnFunc func() *clai
 	return &c.fixedVulns[idx], ok
 }
 
+type ranger struct {
```

Comment:
I think I'm still struggling with figuring out the purpose of this and why it's shared per VEX doc

Reply:
The reason it is scoped to the VEX document is that we need to know the lowest affect range(derived from the fixed in version) for that oci image out of all the entries for that image in the document. I.e. for that vulnerability, what is the lowest fixed in version for image `x`  so we are about to make an infinite lower-bound for the lowest. For example, we extract information like this:
```
OCI     Fixed in version    Range
myimg   4.5.2               4.5.0->4.5.2
myimg   4.4.7               4.4.0->4.4.7
myimg   4.3.12              4.3.0->4.3.12
```
But we don't actually want that we want the last range to be 
```
0.0.0->4.3.12
```
..to address anything before (at least that was the guidance). Hence the function to zero out the lowest- lowest bound.

Reply:
ooooo I see now, thanks